qadence_protocols/mitigations/readout.py

```diff
@@ -14,8 +13,26 @@
 from scipy.optimize import LinearConstraint, minimize
 
 
-def corrected_probas(p_corr: npt.NDArray, T: npt.NDArray, p_raw: npt.NDArray) -> np.double:
-    return norm(T @ p_corr.T - p_raw.T, ord=2) ** 2
+def tensor_rank_mult(qubit_matrices: torch.Tensor, prob_vect: npt.NDArray) -> npt.NDArray:
```

Comment:
I guess a more meaningful name could help and some doctrings. I would also convert `qubit_matrices` as NDArray when called.

Reply:
@rajaiitp can you do these matrix multiplications directly with PyTorch instead?

Reply:
@dominikandreasseitz  I'm using this tensor_rank_mult that PyQ uses in the backed i understand. Do you have a function call that i can readily use to avoid redundancy